harness/determined/util.py

```diff
@@ -293,3 +294,35 @@ def calculate_batch_sizes(
         )
 
     return per_gpu_batch_size, effective_batch_size
+
+
+def check_sshd(peers: List[str], deadline: float, port: int) -> None:
```

Comment:
can you make this function operate on a single peer, and call it in a loop when you want to run it against multiple peers.

That makes it a bit more composable I think, which is good for helper functions.

Reply:
I initially did it by single peer, but I figured maybe the deadline would make more sense in the context of a peer group/rendezvous. But you're right, single peer is probably better for a util function.